### DISCUSSION TITLE
Layer-2 is coming for The Open Network

### DISCUSSION THREAD

**Discussion by serrq** [horizontalsystems/unstoppable-wallet-android#7609] [General]
> The team made the announcement on Tuesday at the Ethereum Community Conference (EthCC) in Brussels, Belgium.
The new protocol, called the TON Applications Chain (TAC), will make use of Polygon’s Chain Development Kit (CDK), a customizable toolkit that lets developers create their own layer-2 blockchains based off of Polygon’s zero-knowledge technology, as well as Polygon’s AggLayer, their interoperability layer for solving blockchain fragmentation.

https://www.coindesk.com/tech/2024/07/09/ton-blockchain-will-release-new-layer-2-based-on-polygons-technology/

https://www.cryptopolitan.com/ton-blockchain-to-use-polygons-tech/

**Comment by esengulov** [comment]
The official reps of TON foundation deny these reports)
